### PR TITLE
post: announcing ngx-speculoos

### PR DESCRIPTION
**Pull request by jnizet**
Scheduled for Wednesday, but can be delayed if it clashes with another blog post.

### CODE REVIEW

File: _posts/2018-05-30-announcing-ngx-speculoos.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Writing Angular unit tests for components quickly leads to quite a lot of boilerplate, and if you're not careful, code duplication and not type-safe code, too. 
+Especially when dealing with forms. 
+
+Out of the frustration from this non-ideal code, we decided to write a small library to help with these issues, and to rely on the page object pattern when it makes sense.
+
+Let me thus introduce [ngx-speculoos](https://ngx-speculoos.ninja-squad.com). 
+
+It's free, as in beer, and as in speach. 
+
+It uses the standard Angular TestBed and ComponentFixture abstractions, so you should get up to speed in a few minutes.
```
Comment (cexbrayat) [inline _posts/2018-05-30-announcing-ngx-speculoos.md]: `TestBed` and `ComponentFixture` (backticks)

Comment (cexbrayat) [reply _posts/2018-05-30-announcing-ngx-speculoos.md]: you should *be* up to speed?

File: _posts/2018-05-30-announcing-ngx-speculoos.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## What's that name?
+
+<div style="float: left;"><img src="/assets/images/speculoos.jpg" alt="speculoos cookies" /></div>
+
+Well, `ngx` stands for *Angular extension*.
+
+Oh, you meant the *other* part of the name?
+
+A *speculoos* is a delicious cookie from Belgium, where one quarter of the Ninja Squad staff (i.e. me) comes from.
```
Comment (cexbrayat) [inline _posts/2018-05-30-announcing-ngx-speculoos.md]: staff -> team ?

File: _posts/2018-05-30-announcing-ngx-speculoos.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+```
+    expect(tester.countryErrors).toContainText('The country is mandatory');
+    expect(tester.city).toBeNull();
+
+    tester.country.selectValue('FR');
+
+    expect(tester.countryErrors).toBeNull();
+    expect(tester.city.optionValues).toEqual(['PARIS', 'LYON']);
+    expect(tester.city.optionLabels).toEqual(['Paris', 'Lyon']);
+    expect(tester.city).toHaveSelectedLabel('Paris');
```
Comment (cexbrayat) [inline _posts/2018-05-30-announcing-ngx-speculoos.md]: Could add a comment here to highlight that there is a custom matcher

File: _posts/2018-05-30-announcing-ngx-speculoos.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+[The project is on Github](https://github.com/Ninja-Squad/ngx-speculoos), so don't hesitate to star the project if you like it, and to request features, improvements or bug fixes, or even to contribute.
+
+## What's that name?
+
+<div style="float: left;"><img src="/assets/images/speculoos.jpg" alt="speculoos cookies" /></div>
+
+Well, `ngx` stands for *Angular extension*.
+
+Oh, you meant the *other* part of the name?
```
Comment (clacote) [inline _posts/2018-05-30-announcing-ngx-speculoos.md]: 👌

**Pull request merged**